<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="user1" failures="1" tests="1" name="runners.ReRunRunner" time="8.045" errors="0" timestamp="2025-04-09T16:50:40 IST" skipped="0">
  <testcase classname="runners.ReRunRunner" name="runScenario" time="8.045">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at stepDefs.BookSearchStepDef.should_display_crime_search_result(BookSearchStepDef.java:45)
at âœ½.Should display crime search result(file:///C:/workspace/BDDSamples/src/test/resources/features/BookSearch.feature:12)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- runners.ReRunRunner -->
